schema.sql

```diff
@@ -1,10 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    id integer AUTO_INCREMENT PRIMARY KEy
+    id integer AUTO_INCREMENT PRIMARY KEY
     name varchar(100)
     date_of_birth date
     escape_attempts integer
     neutered boolean
     weight_kg decimal
+    species varchar(100) DEFAULT 
```

Comment:
- [ ] [Optional] Let's add a new column of species in this way so the table gets updated completely and we follow the best code practice. 

`ALTER TABLE animals ADD species varchar(255);`